nixos/modules/services/continuous-integration/buildbot/master.nix

```
@@ -268,6 +268,7 @@ in {
     systemd.services.buildbot-master = {
       description = "Buildbot Continuous Integration Server.";
       after = [ "network-online.target" ];
+      wants = [ "network-online.target" ];
```

I am using buildbot in 4 deployments and I don't see why it needs the online target in the first place.

```suggestion
      after = [ "network.target" ];
      wants = [ "network.target" ];
```

---

Perhaps. I don't wish to make functional changes in this PR; can we defer this to a later change?